### ISSUE TITLE
Wrong number of arguments in ActiveSupport::Inflector for Rails 3.2.8

### ISSUE THREAD

**Issue by dinhhuydh** [rwz/ember-cli-rails#125]
Hi,

I'm using Rails 3.2.8. After bundle, i can't run the command `rails generate ember-cli:init`. 
The exception is
```
/Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/activesupport-3.2.18/lib/active_support/inflector/inflections.rb:166:in `inflections': wrong number of arguments (1 for $
) (ArgumentError)
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/ember-cli-rails-0.1.11/lib/ember-cli/engine.rb:9:in `block in <class:Engine>'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/initializable.rb:30:in `instance_exec'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/initializable.rb:30:in `run'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/initializable.rb:55:in `block in run_initializers'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/initializable.rb:54:in `each'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/initializable.rb:54:in `run_initializers'
        from /Users/hdinh/code/employment-hero/vendor/ruby/1.9.1/gems/railties-3.2.18/lib/rails/application.rb:136:in `initialize!'
```

Looking at the source
```
if Rails.version > "3.2"
   ActiveSupport::Inflector.inflections :en do |inflect|
     inflect.acronym "CLI"
  end
end
```

IMO, the Rails version should be greater than 3.2.13 because ActiveSupport::Inflector.inflections in Rails 3.2.8 don't have parameter.

**Comment by rwz** [comment]
You're using 0.1.11. This specific bug was fixed in 0.1.12. The current version is 0.1.13.

**Comment by dinhhuydh** [comment]
Thanks. Works fine now.
